test/specs/reference/tests/12.reference_values_edit.js

```diff
@@ -234,6 +234,70 @@ exports.execute = function (options) {
 
             testTupleValidity(6, values);
         });
+        
+        
+        describe("Testing for EDIT JSON and JSONB Values", function() {
+            var expectedValues=[{"id":"1001","json_col":true,"jsonb_col":true},
+            {"id":"1002","json_col":{},"jsonb_col":{}},
+            {"id":"1003","json_col":{"name":"test"},"jsonb_col":{"name":"test"}},
+            {"id":"1004","json_col":false,"jsonb_col":false},
+            {"id":"1005","json_col":2.9,"jsonb_col":2.9},
+            {"id":"1006","json_col":"","jsonb_col":""}];
+
+            var catalog_id = process.env.DEFAULT_CATALOG,
+                schemaName = "reference_schema",
+                tableName = "jsontest_table",
+                lowerLimit = 1001,
+                upperLimit = 2001,
+                limit = 6;
+
+            var multipleEntityUri=options.url + "/catalog/" + catalog_id + "/entity/" + schemaName + ":"+ tableName ;
+
+            var reference, page, tuples;
+
+            beforeAll(function(done) {
+                
+                // Fetch the entities beforehand
+                options.ermRest.resolve(multipleEntityUri).then(function (response) {
+                    reference = response;
```

Comment:
the function call that changes the `context` of a reference is `.contextualize.CONTEXTNAME`. so the reference now is not contextualize. You need to change this to `reference = response.contextualize.entryEdit`